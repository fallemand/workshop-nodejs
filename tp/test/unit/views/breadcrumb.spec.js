// const React = require('react');
// const renderer = require('react-test-renderer');
// const Breadcrumb = require('../../../views/components/Breadcrumb');
//
// describe('Breadcrumb (Snapshot)', () => {
//
//   const category = {
//
//     category: [
//     {
//       id: "MLA5726",
//       name: "Electrodomésticos y Aires Ac."
//     },
//     {
//       id: "MLA10225",
//       name: "Electrodomésticos de Cocina"
//     },
//     {
//       id: "MLA10068",
//       name: "Tostadoras"
//     }
//   ]
//   }
//
//
//   it('Render with query', () => {
//     const component = renderer.create(<Breadcrumb query="iph" category={this.category} />);
//
//     const json = component.toJSON();
//     expect(json).toMatchSnapshot();
//   });
//
//
//   it('Render without query', () => {
//     const component = renderer.create(<Breadcrumb />);
//
//     const json = component.toJSON();
//     expect(json).toMatchSnapshot();
//   });
//
// });
