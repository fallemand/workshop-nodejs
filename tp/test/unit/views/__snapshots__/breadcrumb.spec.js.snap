// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb (Snapshot) Render with query 1`] = `
<ul
  className="breadcrumb"
>
  <li
    className="breadcrumb__element"
  >
    "
    iph
    "
  </li>
</ul>
`;

exports[`Breadcrumb (Snapshot) Render without query 1`] = `
<ul
  className="breadcrumb"
/>
`;
